index.js

```diff
@@ -100,6 +100,10 @@ function photon (imageUrl, opts) {
   return photonUrl;
 }
 
+function isWpcomImage( host ) {
+  return /\.files\.wordpress\.com$/.test(host)
```

Comment:
What about using `endsWith()` instead? (ES6 or lodash, whatever works)

Reply:
I'd rather leave this. `lodash` is not a dependency and it just seems more consistent when we've got something so similar a few lines below:

https://github.com/Automattic/photon.js/blob/462b747f683e5c01cf2fdbb947e6682d35847d6d/index.js#L108

Reply:
Yeah, I'm fine with leaving it as-is too. Maybe just add a trailing semicolon 😉

Reply:
I'm lost without prettier 😮